modules/ROOT/pages/queries-aggregations/filtering.adoc

```diff
@@ -178,34 +212,37 @@ Conversely, the following operator is available on array fields, and accepts a s
 
 These operators are available for all types apart from `Boolean`.
 
-== Logical operators
+== Filtering spatial types
 
-All operators can be combined using the logical operators `AND`, `OR`, and `NOT`.
-They can also be standalone operators, which means that they can be used as such and not be appended to field names.
+Both the `Point` and the `CartesianPoint` types use xref::queries-aggregations/filtering.adoc#_numerical_operators[numerical operators] and also have an additional `_DISTANCE` filter.
+Here is a list of what each filter does for the two types:
 
-These operators accept an array argument with items of the same format as the `where` argument, which means they can also be nested to form complex combinations.
+* `_LT`: checks if a point is less than the distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_LTE`: checks if a point is less than or equal to the distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_DISTANCE`: checks if a point is the exact distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_GT`: checks if a point is greater than the distance in the `distance` field away (in meters) from the point specified by the `point` field.
+* `_GTE`: checks if a point is greater than or equal to the distance in the `distance` field away (in meters) from the point specified by the `point` field.
 
-For example, if you want to match all actors by the name of either "Keanu" or not belonging to the "Pantoliano" family, that played in "The Matrix" movie, here is how you can query that:
+All filters take the following type as an argument:
 
 [source, graphql, indent=0]
 ----
-query {
-    actors(where: { 
-        AND: [
-            { 
-                OR: [
-                    { name_CONTAINS: "Keanu" },
-                    { NOT: { name_ENDS_WITH: "Pantoliano" } }
-                ]
-            },
-            {
-                movies_SOME: { title: "The Matrix" }
-            }
-        ]}
-    ) {
+input PointDistance {
```

Comment:
When `Point` and `CartesianPoint` were unified, the `CartesianPointDistance` is no longer present as well as an example on how to use a filter on `CartesianPoint`

```graphql
input CartesianPointDistance {
    point: CartesianPoint!
    distance: Float!
}
```


Apart from that it looks good to me!

Reply:
added the missing listing -

there was no specific example for the usage of `CartesianPoint`, but maybe the example can be used with both?